### PR TITLE
chore(windows): move cef4delphi to common/windows 🎢

### PR DESCRIPTION
**Pull request by mcdurdin**
@keymanapp-test-bot skip

### CODE REVIEW

File: common/windows/delphi/ext/cef4delphi/Makefile [hunk 1]
```diff
@@ -13,7 +13,7 @@ build: dirs
 
 install:
     $(DEVTOOLS) -ip $(OUTLIB)\CEF4Delphi.bpl
-    $(DEVTOOLS) -ai $(EXT)\cef4delphi\source
+    $(DEVTOOLS) -ai $(KEYMAN_ROOT)\common\windows\delphi\ext\cef4delphi\source
```
Comment (jahorton) [inline common/windows/delphi/ext/cef4delphi/Makefile:16]: I take it that we'll worry about new Desktop makefile defines later?  (As we made new ones for Developer toward this purpose earlier in the 🎢 chain.

Comment (mcdurdin) [reply common/windows/delphi/ext/cef4delphi/Makefile:16]: We probably don't need a `$(COMMON_EXT)` define, I think. Is that what you were expecting here?

Comment (jahorton) [reply common/windows/delphi/ext/cef4delphi/Makefile:16]: For example, #6719 includes a number of changes that would make good use of that `$(COMMON_EXT)` define - for both platforms.

Comment (jahorton) [reply common/windows/delphi/ext/cef4delphi/Makefile:16]: Also, while I missed where in the chain it was introduced, I'd like to point out the existence of `$(COMMON_ROOT)` by the time of #6736.

**Review by jahorton — APPROVED** [review]

**Pull request merged**